### ISSUE TITLE
Can't change icon title

### ISSUE THREAD

**Issue by iyz91** [ionic-team/ionicons#915]
Should be able to change the icon title so that the hover tooltip matches that. Should be as simple as setting via `aria-label=...` or `title=...`

**Comment by asmaa3107** [comment]
yes plz :(

**Comment by ariscol** [comment]
I think the root of the issue here is the same as 2 others for this repo: We'd like control over the tooltip: the option to remove it altogether, as well as to alter the text that appears there.

**Comment by brandyscarney** [comment]
Thanks for the issue, closing this to track in #838.
